css-animations-1/animation-iteration-count-003.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Animations Test: animation-iteration-count - negative value</title>
+<link rel="author" title="Nokia Inc." href="http://www.nokia.com">
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation-name">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation-duration">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation-iteration-count">
+<meta name="flags" content="animated">
+<meta name="assert" content="When animation-iteration-count is set to negative count,
+                             it is invalid and animation will play from beginning to end once.">
```

Comment:
The spec says that "Negative numbers are invalid."

Seems both latest Chrome and Firefox just use the default value instead of the invalid or zero.